### ISSUE TITLE
urlcheck can't always find the host argument when passed an explicit configfile

### ISSUE THREAD

**Issue by nicknovitski** [gaenserich/hostsblock#47]
It can find a url including `http(s)://` just fine:
```
sudo hostsblock-urlcheck -f test/hostsblock.conf http://www.foo.com
[DETAIL] Using configuration file test/hostsblock.conf.
Checking to see if url is blocked or unblocked...

'www.foo.com' NOT BLOCKED/REDIRECTED
	1) Block www.foo.com
	2) Block www.foo.com and delete all whitelist url entries containing www.foo.com
	3) Keep unblocked (default)
1-3 (default: 3)
```
But with `foo.com` or `www.foo.com`:
```
$ sudo hostsblock-urlcheck -f test/hostsblock.conf foo.com
[DETAIL] Using configuration file test/hostsblock.conf.
Checking to see if url is blocked or unblocked...

'-f' NOT BLOCKED/REDIRECTED
	1) Block -f
	2) Block -f and delete all whitelist url entries containing -f
	3) Keep unblocked (default)
1-3 (default: 3):
```

**Comment by pickfire** [comment]
@nicknovitski Thanks for reporthing the problem. I need to check about it before assining to myself.

**Comment by gaenserich** [comment]
Should be fixed here: 84c3b8b

**Comment by nicknovitski** [comment]
Thanks so much!
